opm/simulators/wells/WellAssemble.cpp

```diff
@@ -243,7 +242,9 @@ assembleControlEqInj(const WellState<Scalar, IndexTraits>& well_state,
             break;
         }
         default:
-            throw("Expected WATER, OIL or GAS as type for injectors " + well_.wellEcl().name());
+            OPM_DEFLOG_THROW(std::runtime_error,
+                             fmt::format("Expected WATER, OIL or GAS as type for injectors {}", well_.wellEcl().name()),
```

Comment:
If we're changing this anyway, I think I'd prefer that we fix the plurality mistake as well ("injectorS" ➡️ "injector" as there's just a single well name here).

Reply:
Will do it.